test/e2e/util.go

```diff
@@ -208,6 +209,21 @@ func GetCronJob(namespace, namePrefix string, kubeclient kubernetes.Interface) (
 	return nil, nil
 }
 
+// GetJob retrieves a given cronJob based on namespace, and the cronJob name prefix
+func GetJob(namespace, namePrefix string, kubeclient kubernetes.Interface) (*batchv1.Job, error) {
```

Comment:
Wouldn't it be better to call this function `GetCronJob` ?

Reply:
No this should be GetJob, I accidentally forgot to update the comment to properly explain what the function was doing. I believe it should be good now